rotkehlchen/chain/evm/tokens.py

```diff
@@ -334,3 +382,7 @@ def _get_token_exceptions(self) -> list[ChecksumEvmAddress]:
         Each chain needs to implement any chain-specific exceptions here.
         """
         ...
+
+    def maybe_detect_proxies_tokens(self, addresses: list[ChecksumEvmAddress]) -> None:  # pylint: disable=unused-argument  # noqa: E501
+        """Subclasses may implement this method to detect tokens for proxies"""
+        return None
```

Comment:
EvmTokens is a metaclass so this can be an abstract method

Reply:
Yeah, true, but then it would require all subclasses to implement it which may not be needed. Or do you think that having it as abstract is still better?

Reply:
I think leaving it like this is fine for the exact reason you mentioned. Though we are not sure if all evm chains would require this yet. If they end up doing so then it would make sense to make it abstractmethod and force implementation by all.